src/index.html

```diff
@@ -1,13 +1,74 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+      >
+  </head>
+  <body>
+    <div
+      data-qa="card"
+      class="card"
+      >
+      <div class="card__logo">
+        <img
+        class="card__product"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+        >
```

Comment:
check code styling [recommendations](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) and apply all over the page. (point №14)
```suggestion
        <img
          class="card__product"
          src="images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
        >
```